src/questions/sherlock.yaml

```
@@ -0,0 +1,195 @@
+- uuid: e00d2b19-5495-4ce5-87c2-18b9df7f6658
+  type: SingleChoice
+  content: What is Sherlock?
+  hint: No Hint
```

```suggestion
  hint: NoHint
```

Change at other places also